src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -22,7 +23,8 @@ public static string FrameworkDescription
         {
             get
             {
-                return s_frameworkDescription ?? (s_frameworkDescription = $"{FrameworkName} {typeof(object).GetTypeInfo().Assembly.GetName().Version}");
+                return s_frameworkDescription ?? 
+                    (s_frameworkDescription = $"{FrameworkName} {typeof(object).GetTypeInfo().Assembly.GetCustomAttributes().OfType<AssemblyFileVersionAttribute>().Single().Version}");
```

Comment:
This is very inefficient because of it will create instances of all custom attributes, and then filters them out using Linq.

Is it possible to use  ICustomAttributeProvider.GetCustomAttributes(Type attributeType, bool inherit)  here instead?